starknet_py/tests/e2e/mock/compile_contracts_v2_6.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+MOCK_DIRECTORY=starknet_py/tests/e2e/mock
+CONTRACTS_DIRECTORY="$MOCK_DIRECTORY"/contracts_v2_6
+CONTRACTS_COMPILED_DIRECTORY="$MOCK_DIRECTORY"/contracts_compiled_v2_6
+
+# get path to Cargo.toml
+MANIFEST_PATH=`cat starknet_py/tests/e2e/manifest-path`
+if [ -z "$MANIFEST_PATH" ]
+then
+      echo "File 'manifest-path' was not found in directory 'starknet_py/tests/e2e'. More info here: https://starknetpy.readthedocs.io/en/latest/development.html#setup"
```

Comment:
nitpick: in bash, use 2 spaces instead of 4 for indentation

Reply:
These changes will be added in #1322